packages/runtime/container-runtime/src/channelCollection.ts

```diff
@@ -171,10 +171,6 @@ export function wrapContext(context: IFluidParentContext): IFluidParentContext {
 		getAudience: (...args) => {
 			return context.getAudience(...args);
 		},
-		// back-compat, to be removed in 2.0
-		ensureNoDataModelChanges: (...args) => {
```

Comment:
Need to be deprecated first before removing

Reply:
This is okay @jatgarg because the return type is `IFluidParentContext` which had the deprecation. It is easiest to see what was deprecated or properly in the API reports.